draft-ietf-quic-recovery.md

```diff
@@ -826,16 +826,17 @@ packets.
 ## Persistent Congestion {#persistent-congestion}
 
 When an ACK frame is received that establishes loss of all in-flight packets
-sent over a long enough period of time, the network is considered to be
-experiencing persistent congestion.  Commonly, this can be established by
-consecutive PTOs, but since the PTO timer is reset when a new ack-eliciting
-packet is sent, an explicit duration must be used to account for those cases
-where PTOs do not occur or are substantially delayed. The rationale for this
-threshold is to enable a sender to use initial PTOs for aggressive probing, as
-TCP does with Tail Loss Probe (TLP; see {{RACK}}), before establishing
-persistent congestion, as TCP does with a Retransmission Timeout (RTO; see
-{{?RFC5681}}). The RECOMMENDED value for kPersistentCongestionThreshold is 3,
-which is approximately equivalent to two TLPs before an RTO in TCP.
+sent over a long enough period of time across all the packet number spaces, the
+network is considered to be experiencing persistent congestion.  Commonly, this
+can be established by consecutive PTOs, but since the PTO timer is reset when a
+new ack-eliciting packet is sent, an explicit duration must be used to account
+for those cases where PTOs do not occur or are substantially delayed. The
```

Comment:
This business about establishing PC using PTO is really confusing and I would remove this sentence.

```suggestion
network is considered to be experiencing persistent congestion.  The
```

Reply:
I'm not the editor, but I think I agree. The rationale for choosing 3 as the multiplier is explained below. That's sufficient.